spec/result.rnc

```diff
@@ -0,0 +1,97 @@
+#
+# RELAX NG Schema for SPARQL Query Results XML Format
+#
+# This XML schema is normative.
+# $Id: result.rnc,v 1.1 2007/10/17 14:48:47 eric Exp $
+# 2023-02-16 domel
+#
+#
+
+namespace local = ""
+namespace res = "http://www.w3.org/2005/SPARQL/results#"
```

Comment:
Is the right fix to change 2007 to 2005 or vice versa - I'm quite unclear what is the best way to address the errata.

Reply:
Yes, it was 2007. I have changed to 2005